packages/react-charting/src/types/IDataPoint.ts

```diff
@@ -14,7 +14,7 @@ export interface IBasestate {
   refSelected?: any;
   YValueHover?: { legend?: string; y?: number; color?: string }[];
   hoverYValue?: string | number | null;
-  hoverXValue?: string | number | null;
+  hoverXValue?: string | number | Date | null;
```

Comment:
Since this is a common variable across all charts, are there charts where the Date type won't work.

Reply:
This is not needed anymore.